governance/votes/Executive vote - November 14, 2024.md

```diff
@@ -41,29 +43,52 @@ If this executive proposal does not pass within 30 days, then it will expire and
 
-- Increase the ETH-A Stability Fee by 1 percentage point from 6.25% to **7.25%**.
-- Increase the ETH-B Stability Fee by 1 percentage point from 6.75% to **7.75%**.
-- Increase the ETH-C Stability Fee by 1 percentage point from 6.00% to **7.00%**.
-- Increase the WSTETH-A Stability Fee by 1 percentage point from 7.25% to **8.25%**.
-- Increase the WSTETH-B Stability Fee by 1 percentage point from 7.00% to **8.00%**.
-- Increase the WBTC-A Stability Fee by 1 percentage point from 9.25% to **10.25%**.
-- Increase the WBTC-B Stability Fee by 1 percentage point from 9.75% to **10.75%**.
-- Increase the WBTC-C Stability Fee by 1 percentage point from 9.00% to **10.00%**.
+- Increase the ETH-A Stability Fee by 2 percentage point from 6.25% to **8.25%**.
+- Increase the ETH-B Stability Fee by 2 percentage point from 6.75% to **8.75%**.
+- Increase the ETH-C Stability Fee by 2 percentage point from 6.00% to **8.00%**.
+- Increase the WSTETH-A Stability Fee by 2 percentage point from 7.25% to **9.25%**.
+- Increase the WSTETH-B Stability Fee by 2 percentage point from 7.00% to **9.00%**.
+- Increase the WBTC-A Stability Fee by 2 percentage point from 9.25% to **11.25%**.
+- Increase the WBTC-B Stability Fee by 2 percentage point from 9.75% to **11.75%**.
+- Increase the WBTC-C Stability Fee by 2 percentage point from 9.00% to **11.00%**.
 
 ### Savings Rate Increases
 
-- **Authorization**: [Forum Post](https://forum.sky.money/t/stability-scope-parameter-changes-17-sfs-dsr-ssr-spark-effective-dai-borrow-rate/25522)
-- **Proposal**: [Forum Post](https://forum.sky.money/t/stability-scope-parameter-changes-17-sfs-dsr-ssr-spark-effective-dai-borrow-rate/25522/2)
+- **Authorization**: [Forum Post](https://forum.sky.money/t/stability-scope-parameter-changes-17-sfs-dsr-ssr-spark-effective-dai-borrow-rate/25522/3)
+- **Proposal**: [Forum Post](https://forum.sky.money/t/stability-scope-parameter-changes-17-sfs-dsr-ssr-spark-effective-dai-borrow-rate/25522/4)
 
 If this executive proposal passes, the then following savings rate increases will take place:
 
-- Increase the [Dai Savings Rate (DSR)](https://sky-atlas.powerhouse.io/A.3.2.2.2_Dai_Savings_Rate/8e289e71-7ec5-4ce5-8d4d-41aab7a50f53|57eab733e951) by 1 percentage point from 5.50% to **6.50%**.
-- Increase the [Sky Savings Rate (SSR)](https://sky-atlas.powerhouse.io/A.3.2.2.3_Sky_Savings_Rate/73d91126-21c9-4e8a-bedf-8a51a432bb59|57eab733e951) by 1 percentage point from 6.50% to **7.50%**.
+- Increase the [Dai Savings Rate (DSR)](https://sky-atlas.powerhouse.io/A.3.2.2.2_Dai_Savings_Rate/8e289e71-7ec5-4ce5-8d4d-41aab7a50f53|57eab733e951) by 2 percentage point from 5.50% to **7.50%**.
+- Increase the [Sky Savings Rate (SSR)](https://sky-atlas.powerhouse.io/A.3.2.2.3_Sky_Savings_Rate/73d91126-21c9-4e8a-bedf-8a51a432bb59|57eab733e951) by 2 percentage point from 6.50% to **8.50%**.
+
```

Comment:
Consider: 
```suggestion

Due to how the SparkLend DAI Interest Rate Model (IRM) functions, this will also results in an approximately 2 percentage point increase in the SparkLend Effective DAI Borrow Rate.

```